src/gui/utils.cpp

```diff
@@ -62,6 +63,7 @@ void Utils::Gui::resize(QWidget *widget, const QSize &newSize)
 
 qreal Utils::Gui::screenScalingFactor(const QWidget *widget)
 {
+  return 1.0;
```

Comment:
I think you only need this change in this file, changes on other lines can all be omitted.
Also you should enclose it with:
```c++
#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
    return 1;
#else
    // existing code here
#endif
```